### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 4

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#330] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152324380-d8656a27-8150-4611-97f6-b531ec64b225.png)

**Comment by danieldinella** [comment]
A ⟷ B = 0 solo se A ≠ B.
1. Essendo diversi uno dei due deve essere 0 dunque A ∧ B = 0.
2. Essendo diversi uno dei due deve essere 1 dunque A ∨ B = 1.
3. Essendo diversi abbiamo che A → B = 1 solo se B = 1.
4. Essendo diversi uno dei due sarà 0 e l'altro 1, dunque la proposizione si può scrivere come C ⟷ 0 che sarà vera solo nel caso in cui C = 0.

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/217794725-44c9952f-3769-448a-a350-6f52d1dc7464.png)
